integration-tests/azure-cosmos/README.md

```diff
@@ -0,0 +1,223 @@
+# Azure Cosmos DB sample
+
+This is a sample about implementing REST endpoints using the Quarkus extension to read/write data stored in Azure Cosmos DB.
+
+## Prerequisites
+
+To successfully run this sample, you need:
+
+* JDK 17+ installed with JAVA_HOME configured appropriately
+* Apache Maven 3.8.6+
+* Azure CLI and Azure subscription
+* Docker if you want to build the app as a native executable
+
+You also need to clone the repository and switch to the directory of the sample.
+
+```
+git clone https://github.com/quarkiverse/quarkus-azure-services.git
+cd quarkus-azure-services/integration-tests/azure-cosmos
+```
+
+### Use development iteration version
+
+By default, the sample depends on the development iteration version, which is `999-SNAPSHOT`. To install the development
+iteration version, you need to build it locally.
+
+```
+mvn clean install -DskipTests --file ../../pom.xml
+```
+
+### Use release version
+
+If you want to use the release version, you need to update the version of dependencies in the `pom.xml` file.
+
+First, you need to find out the latest release version of the Quarkus Azure services extensions
+from [releases](https://github.com/quarkiverse/quarkus-azure-services/releases), for example, `1.0.6`.
+
+Then, update the version of dependencies in the `pom.xml` file, for example:
+
+```xml
+<parent>
+    <groupId>io.quarkiverse.azureservices</groupId>
+    <artifactId>quarkus-azure-services-parent</artifactId>
+    <version>1.0.6</version>
+    <relativePath></relativePath>
+</parent>
+```
+
+## Preparing the Azure services
+
+You need to create an Azure Cosmos DB account before running the sample application.
+
+### Logging into Azure
+
+Log into Azure and create a resource group for hosting the Azure Cosmos DB account to be created.
+
+```
+az login
+
+RESOURCE_GROUP_NAME=<resource-group-name>
+az group create \
+    --name ${RESOURCE_GROUP_NAME} \
+    --location westus
+```
+
```

Comment:
This is a suggestion, not a requirement. Given that `.github/create-azure-resources.sh` does a lot of this already, might we want to refactor `.github/create-azure-resources.sh`so that it can serve two purposes

1. Its existing purpose of supporting the workflows and ...
2. A new purpose of performing the setup for interactive running of the integration tests in their role as serving as a sample?

Reply:
Good catch. https://dev.azure.com/azure-redhat-projects/azure-redhat-projects/_workitems/edit/857 is created for tracking the suggestion.